packages/react-expandable-table/src/styles.js

```diff
@@ -0,0 +1,223 @@
+};
+
+export const Header = styled.header`
+  display: flex;
+  align-items: stretch;
+  background-color: ${props => props.theme.colors.gray7};
+  border-bottom: 1px solid ${props => props.theme.selected};
+  border-radius: 2px 2px 0 0;
+  padding: 0 ${props => props.theme.sizes.regular};
+  height: 40px;
+  font-weight: bold;
+
+  ${Container} {
+    font-weight: regular;
+  }
+`;
+Header.defaultProps = {
+  theme: themes.dark,
+};
+
+export const AngleButton = styled.button`
+  all: unset;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transform: rotate(${props => (props.open ? 180 : 0)}deg);
+  transition: transform 0.5s ease-out;
+  ${textStyles('xlarge')};
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  &:focus-visible {
+    box-shadow: 0 0 2px 2px ${props => props.theme.selected};
+  }
+`;
+AngleButton.defaultProps = {
+  theme: themes.dark,
+};
+
+export const Ellipsis = styled(Text)`
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  margin-top: -2px;
+  cursor: pointer;
+  user-select: none;
+  color: ${props =>
+    props.inactive ? props.theme.colors.gray5 : props.theme.primary};
+  &:focus:not(:focus-visible) {
+    outline: 0;
+  }
+`;
+Ellipsis.defaultProps = {
+  theme: themes.dark,
+  tabIndex: -1,
+};
+
+export const InfoIcon = styled(
+  React.forwardRef((props, ref) => (
+    <button type="button" {...props} ref={ref}>
+      <Icon name="question_circle" />
+    </button>
+  ))
+)`
+  all: unset;
+  cursor: pointer;
+  color: ${props => props.theme.colors.gray5};
+  color: ${props => props.theme.prmary};
+  margin-left: ${props => props.theme.sizes.small};
+
+  &:hover,
+  &:focus-visible {
+    color: ${props => props.theme.primary};
+  }
+`;
+InfoIcon.defaultProps = {
+  theme: themes.dark,
+};
+
+const SortAsc = styled(props => <Icon name="sort_asc" {...props} />)``;
+const SortDesc = styled(props => <Icon name="sort_desc" {...props} />)``;
+const ACTIVE_ARROW = props =>
+  props.sort == null ? SortDesc : props.sort === 'desc' ? SortAsc : '.';
+const NEXT_ARROW = props =>
+  props.sort === 'desc' ? SortDesc : props.sort === 'asc' ? SortAsc : '.';
+
+export const SortIcon = styled(
+  React.forwardRef(({ sort, ...props }, ref) => (
+    <button type="button" {...props} ref={ref}>
+      <SortAsc />
+      <SortDesc />
+    </button>
+  ))
+)`
+  all: unset;
+  display: inline-block;
+  position: relative;
+  height: 14px;
+  width: 14px;
+  cursor: pointer;
+  margin-left: ${props => props.theme.sizes.small};
+
+  ${SortAsc}, ${SortDesc} {
+    position: absolute;
+    top: 0;
+    left: 0;
+    color: ${props => props.theme.colors.gray5};
+  }
+
+  ${NEXT_ARROW} {
+    color: ${props => props.theme.selected};
+  }
+
+  &:hover,
+  &:focus-visible {
+    ${ACTIVE_ARROW} {
+      color: ${props => props.theme.primary};
+    }
+  }
+`;
+SortIcon.defaultProps = {
+  theme: themes.dark,
+};
+
+export const TooltipContainer = styled(
```

Comment:
Are we checking the ref type? Not sure if flow raises an error here.

Reply:
I'm not sure I follow, I'm just passing down the `ref` to the `Container` component, because `@quid/react-tooltip` needs to access it.

Reply:
If req is not provided does it throw an error?

Reply:
No

Reply:
If it's not required then ignore my comment.